src/common/modules/ContextMenu.js

```diff
@@ -0,0 +1,26 @@
+import * as Logger from "./Logger.js";
+
+const browserInfo = browser.runtime.getBrowserInfo();
+
+/**
+ * Creates a context menu item
+ * 
+ * @function
```

Comment:
I know some old JSDOC still document it like this in this project here, but I'd rather note that this is public, i.e.:
```suggestion
 * @public
```